ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -151,8 +151,8 @@ ORK_CLASS_AVAILABLE
 
 + (ORKTextAnswerFormat *)textAnswerFormatWithMaximumLength:(NSInteger)maximumLength;
 
-+ (ORKTextAnswerFormat *)textAnswerFormatWithValidationRegex:(NSString *)validationRegex
-                                              invalidMessage:(NSString *)invalidMessage;
++ (ORKTextAnswerFormat *)textAnswerFormatWithValidationRegularExpression:(NSRegularExpression *)validationRegularExpression
```

Comment:
Yeah, I know what deprecation flags are intended for. :) 

However, in this case, I'm inclined to lean on Umer's side.

I see a more reasonable use for deprecation flags on frameworks that are bundled with the *iOS SDK*, where OS updates are greatly encouraged due to severity of security fixes and the scope of the improvements.

On a smaller standalone framework such a *ResearchKit*, developers can keep with their current version if they don't want the hassle of upgrading. This is such a small and trivial change from the API calling point that I continue to favor a shiny and lean API over causing some hassle to developers. :D